homeassistant/components/lektrico/services.yaml

```diff
@@ -0,0 +1,8 @@
+identify:
+  name: Identify
+  description: >-
+    Identify a Lektrico charger.
+  target:
+    entity:
+      integration: charger
+      domain: sensor
```

Comment:
I don't see this service in the code?

Reply:
I removed services.yaml file, thank you.